src/values/dynamic-item-value.js

```diff
@@ -35,5 +54,3 @@ export default class DynamicItemValue {
     return this._values;
   }
 }
-
-ChildElements.includeInto(DynamicItemValue);
```

Comment:
Why was this not workable? I liked this!

Reply:
For this values class, we have some getters (`metadata` and `values`) that are set in the class. Then we have `elements` that was created via this mixin. It is inconsistent to figure out where the various getters comes from. For an actual Element class it does make sense.

Reply:
and here I thought it was an exact use case for composability... lol. I can see how it could be seen as inconsistent though.

Reply:
In my experience mixins like this introduce behavior that is less than obvious to developers and results in time lost figuring out how something works.
Case in point, I was completely oblivious that the functionality I implemented above had already been provided by this mixin (after which the team decided against the mixin). In my opinion, reusable functionality should come from a utility function that is explicitly called in implementations so it's obvious where behavior is coming from.
But that's just my opinion.

Reply:
👍 that all makes sense, inheritance/mixins are definitely information hiding. I agree it’s even extra confusing in JS where that pattern is less common. What was nice was all the other methods. But we could refactor it to a more functional style of utility helpers like you suggested.